equinox-shipping/Domain/FinalizationTransaction.fs

```diff
@@ -0,0 +1,100 @@
+module FinalizationTransaction
+
+open Types
+open FSharp.UMX
+
+let [<Literal>] Category = "FinalizationTransaction"
+
+type State =
+    | Initial
+    | Running   of RunningState
+    | Completed of success : bool
+and RunningState =
+    | Assigning of shipmentIds : string<shipmentId>[]
+    | Assigned  of shipmentIds : string<shipmentId>[]
+    | Reverting of shipmentIds : string<shipmentId>[]
+
+type FinalizationTransactionState = { container : string<containerId> option; state : State }
+
+module Events =
+
+    let streamName (transactionId : string<transactionId>) = FsCodec.StreamName.create Category (UMX.untag transactionId)
+
+    type Event =
+        | FinalizationRequested of {| containerId: string; shipmentIds : string[] |}
+        | AssignmentCompleted   of {| shipmentIds : string[] |}
```

Comment:
if this is same as in FinalizationRequested, why have it? if not, can we have a one liner comment as to what the plan is please?